client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -65,16 +77,7 @@ export const shouldGateStats = ( state: object, siteId: number | null, statType:
 	}
 
 	// check if the site has paid stats feature
-	if ( siteHasPaidStats ) {
-		return false;
-	}
-
-	// check feature flags
-	if ( ! isPaidStatsEnabled ) {
-		return false;
-	}
-	if ( isOdysseyStats ) {
-		// don't gate stats if using Odyssey stats
+	if ( siteHasPaidStats || ! siteFeatures ) {
```

Comment:
Would like to double check `! siteFeatures` here is an indicator that data hasn't been loaded yet?

Reply:
Seems to be null until it's loaded.